src/styles/movieDetails.module.css

```diff
@@ -0,0 +1,145 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 20px auto; /* Center container with equal margin */
+  font-family: Arial, sans-serif;
+  width: 100%;
+  max-width: 1170px; /* Set a max-width for better layout on larger screens */
+  height: auto; /* Adjust height based on content */
+  padding: 20px; /* Add padding for inner spacing */
+  border-radius: 10px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1); /* Add a subtle shadow */
+  background-color: #fff; /* Set a background color */
+  opacity: 1;
+  margin-bottom: 20px;
+  height: 188px;
+}
+
+.movieDetails {
+  display: grid;
+  grid-template-columns: 308.35px 1fr; /* Two columns: one for the poster, one for the trailer and text */
+  grid-template-rows: auto auto; /* Two rows: one for the poster and trailer, one for the text */
+  gap: 20px; /* Add some gap between the grid items */
+  width: 1170px;
+  padding: 20px;
+  margin-inline: auto;
+  background-color: #fff;
+  border-radius: 10px 0px 0px 0px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1); /* Add a subtle shadow */
+}
+
+.movieDetails img {
+  grid-column: 1 / 2; /* Poster in the first column */
+  grid-row: 1 / 2; /* Poster in the first row */
+  width: 308.35px;
+  height: 416px;
+  border-radius: 8px;
+  margin-inline-end: 9px;
+}
+
+.trailer {
+  grid-column: 2 / 3; /* Trailer in the second column */
+  grid-row: 1 / 2; /* Trailer in the first row */
+  width: 100%;
+  height: 416px; /* Match the height of the poster */
+  border-radius: 10px;
+}
+
+.movieDetailsInfo {
+  grid-column: 1 / 3; /* Text spans both columns */
+  grid-row: 2 / 3; /* Text in the second row */
+  display: flex;
+  flex-direction: column;
+  gap: 21px;
+  padding: 10px;
+  text-transform: uppercase;
+  color: rgba(0, 48, 85, 1);
+  size: 16px;
+}
+
+.castCards {
+  list-style: none;
+  display: flex;
+  /* display: grid; */
+  text-align: center;
+  /* grid-template-columns: repeat(3, 1fr); */
+  /* gap: 20px; */
+  /* width: 200%; */
+  width: fit-content;
+}
+
+.actorCard {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+  background-color: #fff;
+  border-radius: 6px;
+  margin: 15px;
+  border-radius: 10px;
+  padding: 12px;
+  text-align: center;
+  font-size: 18px;
+  font-weight: bold;
+  margin-bottom: 5px;
+  display: flex;
+  width: 225px;
+  height: fit-content;
+  justify-content: center;
+}
+
+#image {
+  height: 50px;
+  width: 50px;
+  object-fit: cover;
+}
+
+.span {
+  font-weight: bold;
+}
+
+#overview {
+  margin-bottom: 15px;
+}
+
+#characterName {
+  width: 100%;
+  height: 100%;
+  opacity: 0px;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content:center;
+  display: flex;
+  margin-top: 10px;
+
+}
+
+#characterName h6 {
+  color: rgba(105, 117, 134, 1);
+  font-family: Inter;
+  font-size: 12px;
+} 
+
+#characterName span {
+  width: 100%;
+  height: 17px;
+  gap: 0px;
+  opacity: 0px;
```

Comment:
Opacity is a value 0-1 does not have a unit. This applies to other CSS files in the project.